Wikipedia/Code/ExploreViewController.swift

```diff
@@ -1075,6 +1075,77 @@ extension ExploreViewController {
         
         return true
     }
+    
+    func presentArticleTabsAnnouncement() {
+        guard WMFArticleTabsDataController.shared.needsMoreDynamicTabs,
+              !WMFArticleTabsDataController.shared.hasPresentedTooltips
```

Comment:
According to the ticket, this is not needed

> [not needed anymore] If users have already seen the tooltips, we do not need to show the announcement

There's a `WMFArticleTabsDataController.shared.hasSeenFeatureAnnouncement` check missing - the modal is being presented evrytime you reach the explore feed